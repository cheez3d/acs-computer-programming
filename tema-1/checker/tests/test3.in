Cadoul de CraCiun
17773366444#444087772407777266444206#66677778855588444
dulciuri si acadele sub brad
2
dulciuri fructe
acadele bomboane
salata beof sarmale cozonac
