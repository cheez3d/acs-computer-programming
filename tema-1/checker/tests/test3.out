1222#236668855503#3301222777212224448866
Renii trag sania mosului
3337778822283307777444022666#6226662663307777882202277723
4
77772555282 2233666333 777727776255533 2226669999666662222 
2226669999666662222 4
0
99 96 77 77 77 77 76 72 69 66 66 66 66 63 55 55 33 33 33 28 25 25 22 22 22 22 20 7 2 
